dnf-behave-tests/features/logs.feature

```diff
@@ -26,3 +26,42 @@ Scenario: dnf.rpm.log doesn't contain duplicate entries ()
         .* INFO --- logging initialized ---
         .* SUBDEBUG Erase: flac-1.3.3-3.fc29.x86_64
         """
+
+
+@bz1802074
+Scenario Outline: logfilelevel <level> controls verbosity of logs
+Given I use repository "dnf-ci-fedora-updates"
+  And I configure dnf with
+      | key          | value   |
+      | logfilelevel | <level> |
+  And I execute dnf with args "install flac"
+  And I execute dnf with args "remove flac"
+ When I execute "cat {context.dnf.installroot}/var/log/dnf.log | wc -l"
+ Then stdout is
+      """
+      <dnf_lines>
+      """
+ When I execute "cat {context.dnf.installroot}/var/log/dnf.librepo.log | wc -l"
+ Then stdout is
+      """
+      <librepo_lines>
+      """
+ When I execute "cat {context.dnf.installroot}/var/log/hawkey.log | wc -l"
+ Then stdout is
+      """
+      <hawkey_lines>
+      """
+
+Examples:
+      | level | dnf_lines | librepo_lines | hawkey_lines |
+      | 0     | 0         | 7             | 4            |
+      | 1     | 48        | 7             | 4            |
+      | 2     | 48        | 7             | 4            |
+      | 3     | 64        | 7             | 4            |
+      | 4     | 64        | 7             | 4            |
+      | 5     | 64        | 7             | 4            |
+      | 6     | 64        | 7             | 4            |
+      | 7     | 92        | 7             | 4            |
+      | 8     | 92        | 7             | 4            |
+      | 9     | 92        | 7             | 4            |
+      | 10    | 92        | 71            | 23           |
```

Comment:
Heh... I quite dislike this, because the numbers are going to change slightly on every change to what's logged during the dnf operations used... but at the same time it's a test.. and it's quite cleverly done :slightly_smiling_face: 

Though.. could we check via regex that there's a message with the expected level in the logfile instead? Then the test wouldn't depend on the exact number of lines...

Reply:
ok, I have pushed an update. :slightly_smiling_face: 
I am also thinking about keeping only one level for each unique setting, this way it runs 20 tests in total which seems redundant.

Reply:
Agreed, one test per debug level should be enough.

Reply:
Updated.